src/general/Species.cs

```diff
@@ -119,13 +119,21 @@ public Color GUIColour
     public string FormattedName => Genus + " " + Epithet;
 
     /// <summary>
-    ///   Returns <see cref="FormattedName"/> but includes bbcode tags for styling. Player's species will be emphasized
-    ///   with bolding.
+    ///   Returns <see cref="FormattedName"/> but includes bbcode tags for styling and hover-over tooltips. Player's
+    ///   species will be emphasized with bolding.
     /// </summary>
     [JsonIgnore]
     public string FormattedNameBbCode => PlayerSpecies ?
-        $"[url=species:{ID}][b][i]{FormattedName}[/i][/b][/url]" :
-        $"[url=species:{ID}][i]{FormattedName}[/i][/url]";
+        $"[b][i]{FormattedNameBbCodeUnstyled}[/i][/b]" :
+        $"[i]{FormattedNameBbCodeUnstyled}[/i]";
+
+    /// <summary>
+    ///   Returns <see cref="FormattedName"/> but includes bbcode tags for hover-over tooltips.
+    /// </summary>
+    [JsonIgnore]
+    public string FormattedNameBbCodeUnstyled => PlayerSpecies ?
```

Comment:
Isn't the values same regardless of what PlayerSpecies is here?

Reply:
Right, didn't notice this somehow